gschem/src/x_newtext.c

```diff
@@ -75,7 +75,8 @@ dialog_response_apply (NewText *dialog)
   int align = LOWER_LEFT;
   int color = TEXT_COLOR;
   int rotate = 0;
-  int size = 12;
+  GschemToplevel *w_current = GSCHEM_DIALOG (dialog)->w_current;
```

Comment:
Source files with obsolete naming scheme, e.g. which names start with `x_`, are those having many things to fix. @ehennes775 has fixed many dialogs (adding properties, classes and so on) and renamed them using new naming scheme (e.g. *gschem* code file names start with `gschem_`. You can use his code as an example of refactoring. Now that naming scheme is of course obsolete, too, and we have to come up with a new scheme. I think probably something like e.g. `lepton_multiattrib.c` would be suitable.